src/panels/media-browser/ha-panel-media-browser.ts

```diff
@@ -74,11 +84,29 @@ class PanelMediaBrowser extends LitElement {
     `;
   }
 
+  public updated(changedProps: PropertyValues): void {
+    super.updated(changedProps);
+
+    if (!changedProps.has("route")) {
+      return;
+    }
+
+    if (this.route.path === "") {
```

Comment:
```suggestion
    if (this.route.path === "" || this.route.path === "/") {
```

Reply:
This Safe guards removing everything after the slash but leaving the slash

Should we also check to see if the entityId is valid?

![image](https://user-images.githubusercontent.com/18730868/149184699-ff2e1ca9-5012-4d4d-b14c-af39678d3f0b.png)

Reply:
I think that error being shown is just fine. It means people have messed with the URL.

Reply:
But the error isn't very user-friendly. Example: If you tried to navigate to a page that doesn't exist for some websites then you are redirected to the home page. (there are 404 examples) but its probably better to push them to the entityId we have from the last time they chose a correct one if we can

Reply:
Right, but there is no way we navigate users to pages that don't exist today.